css/styles.css

```diff
@@ -923,20 +943,21 @@ header {
 .footer h3 {
   margin: 20px auto;
   font-family: "work sans", "sans";
-  color: rgb(74, 80, 187);
+  color:  rgb(28, 34, 148);
   text-transform: uppercase;
-  font-size: 1.5rem;
+  font-size: 1.5rem;"Space Mono", monospace
```

Comment:
this is causing the problem

Reply:
yeah yeah just saw it and gonna do it :+1: 
thanks